addons/payment/views/payment_provider_views.xml

```diff
@@ -93,12 +93,22 @@
                             </group>
                         </page>
                         <page string="Fees" name="fees" attrs="{'invisible': [('support_fees', '=', False)]}">
-                            <group name="payment_fees">
+                            <group>
                                 <field name="fees_active"/>
-                                <field name="fees_dom_fixed" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_dom_var" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_int_fixed" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_int_var" attrs="{'invisible': [('fees_active', '=', False)]}"/>
+                            </group>
+                            <group name="payment_fees" attrs="{'invisible': [('fees_active', '=', False)]}">
+                                <group>
+                                    <group string="Extra Fees">
+                                        <field name="fees_dom_fixed"
+                                            options="{'currency_field': 'main_currency_id'}"/>
+                                        <field name="fees_dom_var"
+                                            widget="percentage"/>
+                                        <field name="fees_int_fixed"
+                                            options="{'currency_field': 'main_currency_id'}"/>
```

Comment:
What is weird is that we still need to add this option in order the monetary widget to display the currency symbol, even though the currency is given at the field definition
@AntoineVDV @Feyensv

Reply:
Seems like a framework bug :eyes:

Reply:
Rebase done